### PR TITLE
cv-styled

### PR DESCRIPTION
**Pull request by diesttro**

### CODE REVIEW

File: staff/carlos-diestro/cv-styled/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <img src="https://via.placeholder.com/90x90" alt="John">
+      </div>
+      <div class="name item-2">
+        <h1>John Doe</h1>
+        <h3>Web Developer</h3>
+      </div>
+      <address class="contact item-3">
+        Roc Boronat, 35, 08005 Barcelona
+        <br>
+        <a href="https://github.com" target="_blank">github.com/jdoe</a>
```
Comment (aleixcam) [inline staff/carlos-diestro/cv-styled/index.html:23]: use ```<p>```s for newlines instead of br

File: staff/carlos-diestro/cv-styled/style.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+.name {
+  margin-left: 1em;
+}
+
+.name h1 {
+  margin: 0.5rem 0;
+}
+
+.name h3 {
+  margin: 0.5rem 0;
```
Comment (aleixcam) [inline staff/carlos-diestro/cv-styled/style.css:58]: Repeated style, may you want to use ```.name h1, .name h3 {```

Comment (diesttro) [reply staff/carlos-diestro/cv-styled/style.css:58]: You're right.

File: staff/carlos-diestro/cv-styled/index.html (hunk 1 above)
Comment (diesttro) [reply staff/carlos-diestro/cv-styled/index.html:23]: In this case it's fine. More info here https://developer.mozilla.org/es/docs/Web/HTML/Elemento/br

**Pull request merged**